src/components/Table/Table.tsx

```diff
@@ -121,8 +78,11 @@ export function Table<T extends { id: number }>(props: TableProps<T>) {
     return sortedData;
   }, [filtersState, sortState, props.data]);
 
-  console.log(`filterState`, filtersState);
-  console.log(`sortState`, sortState);
+  const displaySortingIcon = (sortingStatus: string | null) => {
+    if (sortingStatus === "desc") return <IconSortDescending />;
+    if (sortingStatus === "asc") return <IconSortAscending />;
+    if (sortingStatus === null) return <div></div>;
```

Comment:
1. use `else if` if the ifs are related
2. Makes more sense to do:

```
if ('desc') return ...
else if ('asc') return ...

return null;
```
3. `return null` instead of `<div></div>`